site_scons/components/__init__.py

```diff
@@ -132,20 +111,12 @@ def define_mercury(reqs):
                 commands=['./autogen.sh',
                           './configure --prefix=$OFI_PREFIX ' +
                           '--disable-efa ' +
+                          '--disable-psm3 ' +
                           '--without-gdrcopy ' +
```

Comment:
if we decide to remove psm2 support then for now (until we update to most recent OFI release) we need to explicitly disable it via -disable-psm2. most providers in OFI today are compiled as 'auto', where they auto-detect if they can be compiled, and if so, they get enabled.  
The change that landed recnetly to OFI that we requested adds new --enable-only option to OFI which then disables all providers by default except the ones we explicitly enable via --enable-[provider] flag